tests/prediction/time_series/data_loader/transfer_learning_data_loader_test.py

```diff
@@ -77,3 +81,42 @@ def test_train_observed_do_not_change_when_called_multiple_times():
     loader.load_data()
     after = loader.train_observed
     assert np.array_equal(before, after)
+
+
+def test_x_test_data_is_exported_when_export_path_is_provided():
+    EXPORT_PATH = "tests/prediction/time_series/test_data/export"
+    cfg = data_config.DataConfig(
+        synthetic="/tests/prediction/time_series/test_data/num_species_1/simulated/",
+        observed="/tests/prediction/time_series/test_data/num_species_1/real/",
+        time_series=time_series_config.TimeSeriesConfig(
+            input_features=["Infected", "Recovered"],
+            output_features=["Infected", "Recovered"],
+        ),
+        export_path=EXPORT_PATH,
+    )
+    loader = transfer_learning_data_loader.TransferLearningDataLoader(cfg)
+    _ = loader.X_test
```

Comment:
Why do we need this assignment?

Reply:
I thought this is the common way to call a method that returns sth we don't want to further use in the test. It appears, you can simply not access the return value at all. Changed accordingly.

Reply:
I thought this is the common way to call a method that returns sth we don't want to further use in the test. It appears, you can simply not access the return value at all. Changed accordingly.